[kubernetes-cluster-api] initial integration

AdamKorcz opened the pull request.

AdamKorcz commented:
@jonathanmetzman @oliverchang Cluster API is ready for review.

randomvariable commented:
Do we need to add some folk to the auto_ccs ?

randomvariable commented:
I'm naadir [at] randomvariable.co.uk (is a google account)

randomvariable commented:
cc @fabriziopandini @sbueringer @CecileRobertMichon  @enxebre @vincepri

randomvariable commented:
Other than that, no further comments as don't know enough about the mechanics atm.

AdamKorcz commented:
> Do we need to add some folk to the auto_ccs ?

Yes, we have added a few maintainers. It is not crucial to get everyone added at this stage. The cc list can be modified at any time after integration by creating a PR here on OSS-fuzz with changes to the `project.yaml` file.

> I'm naadir [at] randomvariable.co.uk (is a google account)

Thanks, added!

sbueringer commented:
@AdamKorcz

Can you add me too, please? buringerst (at) vmware.com

AdamKorcz commented:
> Can you add me too, please? buringerst (at) vmware.com

Added

Ankitasw commented:
I am ok with being added now or post PR merge.
swamyan (at) vmware.com

AdamKorcz commented:
> I am ok with being added now or post PR merge. swamyan (at) vmware.com

Added

jonathanmetzman reviewed (approved):
lgtm
